app/views/collections/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(collection) do |f| %>
+  <% if collection.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(collection.errors.count, "error") %> prohibited this collection from being saved:</h2>
+
+      <ul>
+      <% collection.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+  <div class="form-group row">
```

Comment:
I setup project to use Bootstrap 4 (why start a new project on an old version). AFAICT there wasn't support for 4 yet.